src/binary/CMakeLists.txt

```diff
@@ -99,7 +99,10 @@ target_link_libraries( builtinx ${YAST_YCP_LIBRARY} )
 target_link_libraries( builtinx ${YAST_PLUGIN_SCR_LIBRARY} )
 target_link_libraries( builtinx ${YAST_PLUGIN_WFM_LIBRARY} )
 target_link_libraries( builtinx ${YAST_PLUGIN_UI_LIBRARY} )
-target_link_libraries( builtinx crypt )
+find_library( CRYPT_LIBRARY crypt )
+if ( CRYPT_LIBRARY )
+  target_link_libraries( builtinx ${CRYPT_LIBRARY} )
+endif()
```

Comment:
to be honest, I try to touch cmake as less as possible. So I just tried to fix warning. I can revert it.